raiden/network/proxies/token.py

```diff
@@ -57,8 +57,10 @@ def approve(self, allowed_address, allowance):
         }
         log.debug('approve called', **log_details)
 
+        startgas = 100000
```

Comment:
Is this to make sure that it works with different token implementations? If so please make it a constant

Reply:
Humm, this is actually I bug. I should not have set it a constant value in the proxy only the tests. Will fix.